viamonvifdiscovery/service.go

```diff
@@ -20,6 +20,7 @@ import (
 var (
 	Model             = viamrtsp.Family.WithModel("onvif")
 	errNoCamerasFound = errors.New("no cameras found, ensure cameras are working or check credentials")
+	emptyCred         = device.Credentials{}
```

Comment:
not blocking, but add test?

Reply:
I currently cannot test this, since I need to be able to add camera mocks to these tests. 

Once the circular dependencies can be removed(after discovercomponents is deleted) we can move the discovery service into the `viamonvif` folder. then I can use the camera mocks defined in those tests and add a way to use them with onvif discovery

Reply:
Right, thanks for bringing that up again.